testing/library/utils.go

```diff
@@ -66,11 +66,10 @@ func GetModulesImagesTags(modulePath string) (map[string]interface{}, error) {
 		search = true
 	}
 
+	var err error
 	if search {
-		modulesTags = tags
+		modulesTags = DefaultImagesTags
```

Comment:
After this change, do we still need `init`?

Reply:
Sure, we don't. Removed